_layouts/default.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: "en" }}">
+
+  {%- include head.html -%}
+
+  <body>
+
+    {%- include header.html -%}
```

Comment:
Are header.html and footer.html generated somehow or planned to be added? I don't see them.

Reply:
The site uses minima layout so everything starts from their content, unless we explicitly override it. Those files would come from minima at this time.

https://github.com/jekyll/minima/